js/src/wp-seo-replacevar-plugin.js

```diff
@@ -36,9 +36,7 @@ var ReplaceVar = require( "./values/replaceVar" );
 		this.registerEvents();
 	};
 
-	/**
-	 * GENERIC
-	 */
+	/* GENERIC */
```

Comment:
As you've converted them to single-line comments, they should be just formatted as // ...

Though the initial intent was to separate blocks, thus having a block that stands out more is preferred.

The only thing that had to be adjust was converting from:

/**
 * SPECIALIZED REPLACES		
 */
to

/*
 * SPECIALIZED REPLACES		
 */